Assignment1/Assignment1/CalculateSimpleInterest.m

```diff
@@ -0,0 +1,21 @@
+
+
+#import "CalculateSimpleInterest.h"
+
+@implementation CalculateSimpleInterest
+
+
+
+@synthesize rate;
+@synthesize timePeriod;
+
+float const simpleInterestDivisor=100;
```

Comment:
Notice here thing that you have declared "float const simpleInterestDivisor" in .h file already . and here you are redeclaring it but when you used it in your method , since its private property in .h file, it will first consider property from there (for that you don't even need to synthesize) so here when that value passed here it takes constant initial value which is 0 by default caused your result divided by zero --> inf which means infinity .